mkdocs_monorepo_plugin/parser.py

```diff
@@ -41,15 +43,26 @@ def __loadAliasesAndResolvedPaths(self, nav=None):
             if type(item) is str:
                 value = str
             elif type(item) is dict:
+                key = list(item.keys())[0]
                 value = list(item.values())[0]
+                if key.startswith(WILDCARD_INCLUDE_STATEMENT):
+                    base_path, *name = key[len(WILDCARD_INCLUDE_STATEMENT):].split()
+                    key = " ".join(name)
+                    dirs = sorted(glob.glob(base_path))
```

Comment:
Just noting that I attempted to get recursive /**/ working 
```suggestion
                    dirs = sorted(glob.glob(base_path, recursive=True))
```

And it processes fine, but the nav resolver code that currently exists would need to be modified to support recursive inclusions.

Reply:
Is it worth it to implement the recursive stuff? My thought is this won't apply to any use case we have that I'm aware of and for slightly more complicated structures you could get pretty far with multiple wildcard include statements.

Reply:
@TeamTeaTime I don't think it's worth it *right now*. Possibly in the future, especially since it'll take some more customization to get it working. I'm cool with shipping current functionality.

Reply:
I added support for recursive inclusions and it avoids directories without a mkdocs.yml file in them.